CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 You can find and compare releases at the [GitHub release page](https://github.com/nuwave/lighthouse/releases).
 
 ## Unreleased
-
+- Fix the bug https://github.com/nuwave/lighthouse/issues/2354
```

Comment:
`Fix the bug` says nothing, please describe what the change actually does.

Reply:
Prevent automatic paginator type change if cache directive exists (https://github.com/nuwave/lighthouse/issues/2354).